start

```diff
@@ -11,10 +11,34 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-# Keep this aligned with the Dockerfile at the root of the repository.
-docker run \
-  -v "$(pwd)"/docs:/home/node/app/docs \
-  -v "$(pwd)"/public/images:/home/node/app/packages/preview/public/images \
-  -p 3000:3000 \
-  -p 5001:5001 \
-  qiskit/documentation
+import subprocess
+from pathlib import Path
+
+PWD = Path(__file__).parent
+
+
+def main() -> None:
+    subprocess.run(
+        # Keep this aligned with the Dockerfile at the root of the repository.
+        [
+            "docker",
+            "run",
+            "-v",
+            f"{PWD}/docs:/home/node/app/docs",
+            "-v",
+            f"{PWD}/public/images:/home/node/app/packages/preview/public/images",
+            "-p",
+            "3000:3000",
+            "-p",
+            "5001:5001",
+            "qiskit/documentation",
+        ],
+        check=True,
+    )
+
+
+if __name__ == "__main__":
+    try:
+        main()
+    except KeyboardInterrupt:
+        pass
```

Comment:
Otherwise you get an annoying error message when using ctrl-c to close the server.